shared/actions/favorite.js

```diff
@@ -20,6 +20,10 @@ export function favoriteList (): (dispatch: Dispatch) => void {
           return
         }
 
+        if (!folders) {
```

Comment:
we don't allow the action to send up null, we use it as a proxy for not having requested the favorites